--- xs/InputManager.xs-orig     2007-12-10 23:42:48.000000000 +0100
+++ xs/InputManager.xs  2007-12-10 23:42:07.000000000 +0100
@@ -103,10 +103,10 @@
 InputManager::inputSystemName()
 
 int
-InputManager::numJoysticks()
+InputManager::numJoySticks()
 
 int
 InputManager::numMice()
 
 int
-InputManager::numKeyBoards()
+InputManager::numKeyboards()
...skipping...
--- xs/InputManager.xs-orig     2007-12-10 23:42:48.000000000 +0100
+++ xs/InputManager.xs  2007-12-10 23:42:07.000000000 +0100
@@ -103,10 +103,10 @@
 InputManager::inputSystemName()
 
 int
-InputManager::numJoysticks()
+InputManager::numJoySticks()
 
 int
 InputManager::numMice()
 
 int
-InputManager::numKeyBoards()
+InputManager::numKeyboards()
